Sources/Build/PkgConfig.swift

```diff
@@ -86,23 +85,46 @@ struct PkgConfigParser {
     }
     
     mutating func parse() throws {
+        
+        func removeComment(line: String) -> String {
+            if let commentIndex = line.characters.index(of: "#") {
+                return line[line.characters.startIndex..<commentIndex]
+            }
+            return line
+        }
+        
         let file = File(path: self.pcFile)
         for line in try file.enumerate() {
-            if !line.characters.contains(":") && line.characters.contains("=")  {
-                let equalsIndex = line.characters.index(of: "=")!
+            // Ignore any commented line.
+            if line.hasPrefix("#") || line.isEmpty { continue }
+            // Remove any trailing comment from the line.
+            let line = removeComment(line: line)
```

Comment:
This should probably also ignore lines which are whitespace.

If you swap the order of these, two tests, then you can avoid the duplicated check for `#`.